.github/workflows/createrelease.yml

```diff
@@ -0,0 +1,39 @@
+on:
```

Comment:
Is this an artifact of rebase?

Reply:
My master was just this commit ahead of upstream/master. If you create a new Action they always end up in master. I will disable it in upstream straight after merge, ok?

Reply:
I see. What are the reasons to have master in the fork ahead of the upstream master? Is that because GitHub release actions are not working from branches?

Reply:
At least that is what I experienced. I  created a branch for the actions, switched to that branch in the web interface, created the action, it ended up in master... This was not what I expected...

Reply:
forked master should not be ahead of upstream, I totally agree on that. I'm still learning all those GitHub features, so I don't want to rule out I made a mistake somewhere with these Actions... If I disable it right after merge it will not interfere with anything, though.

Reply:
Or should we just delete the file here?

Reply:
If file is not in upstream master, then deleting it in PR will:
1. make PR specific to the change (no other unrelated changes)
1. safe to merge (if file is in upstream master, merge may delete it AFAIK)